src/zzz_od/application/game_config_checker/predefined_team_checker/predefined_team_checker.py

```
@@ -70,11 +69,11 @@ def check_team_members(self) -> OperationRoundResult:
             return self.round_success()
 
     def update_team_members(self, screen: MatLike) -> None:
-        result_team_list: List[TeamWrapper]
+        result_team_list: list[TeamWrapper]
```

Ruff (0.14.14)

[error] Local variable `result_team_list` is annotated but never used

(F842)